core/src/cg/group4/view/screen_mechanics/ScreenStore.java

```diff
@@ -60,10 +60,10 @@ public static ScreenStore getInstance() {
      */
     public void init() {
         addScreen("Home", new HomeScreen());
-        addScreen("Collection", new CollectiblesScreen());
```

Comment:
This is something that has to be fixed in general. Screens do not properly work with the remote server yet. They now have a hard dependency on the remote server, but what if the player has no WIFI? Screens have to be fixed in another branch.